templates/gmsim/20.4.1.1/root_defaults.yaml

```diff
@@ -0,0 +1,20 @@
+hf:
+  version: 5.4.5.3
+  dt: 0.005
+  rvfac: 0.8
+  sdrop: 50
+  path_dur: 11
+  kappa: 0.045
+  rayset: 1
+bb:
+  fmin: 0.5
+  fmidbot: 1.0
+  no-lf-amp: True
+  dt: 0.005
+emod3d:
+   emod3d_version: 3.0.8
+flo: 1.0
+dt: 0.005
+pSA_periods: [0.01, 0.02, 0.03, 0.04, 0.05, 0.075, 0.1, 0.12, 0.15, 0.17, 0.2, 0.25, 0.3, 0.4, 0.5, 0.6, 0.7, 0.75, 0.8, 0.9, 1.0, 1.25, 1.5, 2.0, 2.5, 3.0, 4.0, 5.0, 6, 7.5, 10.0]
```

Comment:
Needs an im entry as per https://github.com/ucgmsim/slurm_gm_workflow/blob/master/templates/gmsim/20.4.1.2/root_defaults.yaml

Reply:
This line is now deprecated